lib/JsonpMainTemplatePlugin.js

```diff
@@ -141,7 +150,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 						"}"
 					]),
 					"}",
-					"if(parentJsonpFunction) parentJsonpFunction(chunkIds, moreModules, executeModules);",
+					"if(parentJsonpFunction) parentJsonpFunction(data);",
 					"while(resolves.length)",
 					this.indent("resolves.shift()();"),
 					this.entryPointInChildren(chunk) ? [
```

Comment:
If loading with script async, can't this potentially execute the entry point before all sync dependencies are available? (CommonsChunkPlugin et al)